schemata/user.yaml

```diff
@@ -17,4 +17,16 @@ properties:
   avatar:
     type: string
     example: "http://example.net/user.png"
-required: [uuid, name]
+  canUse:
```

Comment:
These need to be under a `properties` key, like in https://github.com/the-grid/apidocs/blob/f859abf63ad4fc3cbacc48228a56d5f3f62def6f/schemata/siteconfig.yaml#L32